riscv_isac/coverage.py

```diff
@@ -328,6 +326,54 @@ def __init__(self, xlen, flen):
         self.swCnt = 0
         self.swFlag = True
 
+#######################################################################################################
+#This function expands the rsval and defining the respective sign, exponent and mantissa correspondence
+#Defining these globally would aid the eval function with these parameters given below
+#######################################################################################################
+def define_sem(flen, rsval, postfix):
+    global fs1, fs2, fs3, fe1, fe2, fe3, fm1, fm2, fm3
```

Comment:
Defining them as global is not a scalable solution. Please refer to the way its done in `simd_val_unpack`